projects/linuxkit/linuxkit/Makefile

```diff
@@ -91,7 +91,12 @@ $(OUTPUT_BIN_DIR)/linux-%/rngd: EXTRA_GO_LDFLAGS=-extldflags -static
 $(OUTPUT_BIN_DIR)/linux-%/rngd: EXTRA_GOBUILD_FLAGS=-tags netgo,osusergo,static_build
 $(OUTPUT_DIR)/rngd/attribution/go-license.csv: CGO_CREATE_BINARIES=true
 
-ca-certificates/images/% dhcpcd/images/% firmware/images/% getty/images/% modprobe/images/% openntpd/images/%: IMAGE_CONTEXT_DIR=$(REPO)/pkg/$(IMAGE_NAME)
+ca-certificates/images/% firmware/images/% getty/images/% modprobe/images/%: IMAGE_CONTEXT_DIR=$(REPO)/pkg/$(IMAGE_NAME)
+
+dhcpcd/images/% openntpd/images/%: BASE_IMAGE_NAME=eks-distro-minimal-base-glibc
+
+$(call IMAGE_TARGETS_FOR_NAME,openntpd): openntpd-useradd/images/export
+openntpd-useradd/images/export: IMAGE_USERADD_USER_NAME=_ntp
```

Comment:
Where is this username coming from? Couldn't find it in the upstream Linuxkit openntpd Dockerfile.